modules/wallet/transactionbuilder.go

```diff
@@ -678,6 +697,21 @@ outer:
 		}
 	}
 
+	// add unconfirmed outputs
+	for _, pt := range w.unconfirmedProcessedTransactions {
+		for _, o := range pt.Outputs {
+			if o.WalletAddress {
+				outputs = append(outputs, modules.SpendableOutput{
+					FundType:           types.SpecifierSiacoinOutput,
+					ID:                 o.ID,
+					UnlockHash:         o.RelatedAddress,
+					Value:              o.Value,
+					ConfirmationHeight: types.BlockHeight(math.MaxUint64), // unconfirmed
```

Comment:
not sure what the best approach is here. I guess this is an argument for making the field `Confirmations` instead of `ConfirmationHeight`; then we could set `Confirmations: 0`. If we instead set `ConfirmationHeight` to some special value, the API client has to do special processing to calculate the number of confirmations.